zadanie.py

```diff
@@ -0,0 +1,30 @@
+foo = []
```

Comment:
imports or documentation should be at the beginning of the file (or file encoding)